canonical-web-and-design/snapcraft.io
# New category page

Fixes https://github.com/canonicalltd/snap-squad/issues/970

## Does not:
Implement infinite scroll - do we want to do this? Really? Do we?

## To consider:
There are 20 featured snaps per category, which leaves a hanging snap - what should we do about that?

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/store
- Click through the "See more" buttons

## Note
- Featured category should not be limited to 20 featured snaps (all the snaps are featured)
